xmtp_db/migrations/2025-08-20-174800_d14n_originator_identity_updates/down.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE identity_updates
```

Comment:
it may be superfluous to add an originator column for identity updates, but wasn't entirely sure what to do and wanted to be safe